index.html

```diff
@@ -36,6 +36,136 @@ <h1 class="heading-primary">I’m Jubaer. <span class="heading-primary__sapan">G
+                        </div>
+                        <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="card-language">
+                            <li class="card-language__item"><a href="#" class="btn-small">Html</a></li>
+                            <li class="card-language__item"><a href="#" class="btn-small">Css</a></li>
+                            <li class="card-language__item"><a href="#" class="btn-small">javascript</a></li>   
+                        </ul>
+                        <a href="#" class="card-cta">See project</a>
+                    </div>
+
+                    <div class="card">
+                        <img src="./images/card-4.jpg" alt="project-4" class="card-img">
+                        <h2 class="card-title">Multi-Post Stories</h2>
+                        <div class="card-description">
+                            <span class="card-description__stack">CANOPY</span>
+                            <div class="card__gray-dot"></div>
+                            <span class="card-description__tec">Back End Dev</span>
+                            <span class="card__gray-dot"></span>
+                            <span class="card-description__year">2015</span>
+                        </div>
+                        <p class="card-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul class="card-language">
+                            <li class="card-language__item"><a href="#" class="btn-small">Html</a></li>
+                            <li class="card-language__item"><a href="#" class="btn-small">Css</a></li>
+                            <li class="card-language__item"><a href="#" class="btn-small">javascript</a></li>   
+                        </ul>
+                        <a href="#" class="card-cta">See project</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="section-about-me">
+            <div class="container">
+                <h2 class="heading-primary">About Myself</h2>
+                <p class="section-hero__paragraph">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <a href="#" class="section-hero__link">LET’S CONNECT</a>
+                <div class="section-hero__social-media">
+                    <img src="./images/twiter.svg" alt="twiter logo" class="twiter-logo">
+                    <img src="./images/in.svg" alt="linked-in logo">
+                    <img src="./images/m.svg" alt="m logo">
+                    <img src="./images/victory.svg" alt="victory logo">
+                    <img src="./images/github.svg" alt="github logo">
+                </div>
```

Comment:
- To help improve the semantics of your site, I suggest wrapping this section in a `ul` tag and each of the images in a corresponding `li` tag. Please fix this 🙏